web/package.json

```diff
@@ -18,7 +18,8 @@
         "check": "run-p type lint coverage",
         "type": "tsc",
         "lint": "eslint .",
-        "format": "eslint --fix .",
+        "fix": "eslint --fix .",
+        "format": "prettier --write .",
```

Comment:
Not 100% sure but I think we are not yet using `prettier` as we don't have a prettier config file for this repository.

Reply:
Also, we could add the `lint` or `fix` command in the husky pre-commit file to double-check on the files edited before a commit is made incase linting on "file change" or "file save" is not enabled or working for an individual.

Reply:
Hi @mulengawilfred we're going to use prettier for format. It now has been setup in vscode settings file, please let me know if you think we need any prettier config.
For husky pre-commit lint check i think we already have that in .husky/pre-commit and i remember it works, could you help double check the settings?

Reply:
Thanks Shaun. the `format` command works for me. After some thought, since we have a shared ESlint config, makes sense to use that than prettier. But I think it's okay for now. 

I see the `lint-staged` command in the husky `pre-commit` file. Thanks for confirming. I've checked as well.